communitiesuk/funding-service-design-frontend
# FLS-778 Redirect if deadline passes whilst form is open

Added flag to redirect the user to the applicant dashboard if the deadline passes whilst form is open.

https://mhclgdigital.atlassian.net/browse/FLS-778


### Change description
If the user opened the form before the deadline but tried to submit the form after the deadline passed they would be taken to this screen
![image](https://github.com/user-attachments/assets/871daf87-493c-4334-83fe-a275c2c4932d)

The change now takes the user to this screen so that they can see the window closed message
![image](https://github.com/user-attachments/assets/6eda5dd6-4bae-434e-8611-533ce79f43ed)



### How to test
- Complete an application but do not submit
- In the DB change the deadline date for the application to a date in the past
- Submit the application
- The user should now be taken to the 'applicant dashboard' so that the user can see the window closed message

### Screenshots of UI changes (if applicable)

The user should now be taken here
![image](https://github.com/user-attachments/assets/6eda5dd6-4bae-434e-8611-533ce79f43ed)
